test/Microsoft.AspNet.Mvc.Xml.Test/XmlAssertTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    public class XmlAssertTest
```

Comment:
don't see tests of
- `<a>hello<b>world</b>hello</a>` compared to `<a>hello<b>world</b>goodbye</a>` (different 2nd text node)
- `<a><b>hello</b<b>world</b></a>` compared to `<a><b>hello</b<b>goodbye</b></a>` (different text in second element)

also, realized you're using "Validates" for both tests that `throw` and don't.  suggest
- `Throws_WithMismatchedTextNodes`
- `ReturnsSuccessfully_WithEmptyElements`
- `Throws_WithMismatchedXmlDeclarations`
- `ReturnsSuccessfully_WithMatchingXmlDeclaration`
- `ReturnsSuccessfully_WithMatchingXmlDeclaration_IgnoringCase`
- `ReturnsSuccessfully_WithMatchingContent_IgnoringAttributeOrder`
- `Throws_WithMismatchedAttributeValues_ReorderingAttributes`

or use the scenario_expectation order e.g. `MismatchedAttributeValues_Throws_ReorderingAttributes`